backup-cron.sh

```
@@ -0,0 +1,135 @@
+            log "ERROR: Backup already running (PID: $OLD_PID), skipping"
+            exit 0
+        else
+            log "WARNING: Stale lock file found, removing"
+            rm -f "$LOCK_FILE"
+        fi
+    fi
+fi
+
+# Create lock file with our PID
+echo $$ > "$LOCK_FILE"
+trap "rm -f $LOCK_FILE" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)